tests/prepare/feature/crb/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+. ../../../images.sh || exit 1
+
+rlJournalStart "CRB Feature Test"
+    rlPhaseStartSetup
+        rlRun "PROVISION_HOW=${PROVISION_HOW:-container}"
+
+        build_container_image "centos/stream9/upstream\:latest"
+        build_container_image "ubi/8/upstream\:latest"
+
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    images="$TEST_IMAGE_PREFIX/centos/stream9/upstream:latest $TEST_IMAGE_PREFIX/ubi/8/upstream:latest ubi9"
+
+    # CRB
+    for image in $images; do
+        # Run the '/crb/enabled' plan, overriding the provision image.
+        # The plan itself contains the check (dnf repolist enabled | grep ...).
+        # Expecting tmt run to succeed (exit code 0) as the check should pass.
+        rlPhaseStartTest "Test /crb/enabled on $image"
+            rlRun "tmt run --all provision --how container --image $image prepare execute plan --name /crb/enabled" 0 "Run /crb/enabled plan for $image"
```

Comment:
```suggestion
            rlRun "tmt run --all provision --how container --image $image plan --name /crb/enabled" 0 "Run /crb/enabled plan for $image"
```

When `--all` is used we don't need to explicitly list `prepare` and `execute` steps.